pkg/cmd/cli/cmd/rollout/latest.go

```diff
@@ -0,0 +1,161 @@
+package rollout
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/api/meta"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	deployutil "github.com/openshift/origin/pkg/deploy/util"
+)
+
+const (
+	rolloutLatestLong = `
+Start a new rollout for a deployment config with the latest state from its triggers
+
+This command is appropriate for running manual rollouts. If you want full control over
+running new rollouts, use "oc set triggers --manual" to disable all triggers in your
```

Comment:
we need a better definition of what does "rollout" mean (I don't think we do have definition of that in docs)